html/Axit/js/validation/banner-validation/banner-form-validation.js

```diff
@@ -0,0 +1,39 @@
+import {checkName, checkEmail, checkPassword} from '../helper-functions.js';
+import {components} from './banner-components.js';
+
+export function addBannerFormValidation() {
+  components.then((response) => {
+    const {
+      bannerForm,
+      errorMessages,
+      nameInput,
+      emailInput,
+      passwordInput,
+    } = response;
+
+    bannerForm.addEventListener('submit', () => {
+      event.preventDefault();
+      validateBannerForm();
+    });
+
+    function validateBannerForm() {
+      let result = [];
+
+      // Name validation
+      result.push(checkName(nameInput, errorMessages[0]));
```

Comment:
hardcoded values is not very good practice, what if you will have more elements? or they change places for some design reasons